gtfs-realtime-validator-lib/pom.xml

```diff
@@ -7,11 +7,11 @@
     <parent>
         <groupId>edu.usf.cutr</groupId>
```

Comment:
If MobilityData is going to be publishing these artifacts going forward, this (and other `groupId` references in pom.xml for this project artifacts) should change to `org.mobilitydata`. This matters for publishing the GitHub packages - see https://docs.github.com/en/actions/publishing-packages/publishing-java-packages-with-maven#about-package-configuration.

Reply:
Good catch!